src/doc_builder/commands/build.py

```diff
@@ -82,7 +82,8 @@ def build_command(args):
     if args.version is None and "dev" not in version:
         doc_config = get_doc_config()
         version_prefix = getattr(doc_config, "version_prefix", "v")
-        version_tag = f"{version_prefix}{version}"
```

Comment:
I can't comment above but at this stage `version` won't have `dev` in it anymore if it's the main version. The test a line 82 should become `version != default_version`